common-lib/src/main/scala/com/gu/mediaservice/lib/ImageStorage.scala

```diff
@@ -34,7 +34,8 @@ trait ImageStorage {
   /** Store a copy of the given file and return the URI of that copy.
     * The file can safely be deleted afterwards.
     */
-  def storeImage(bucket: String, id: String, file: File, mimeType: Option[MimeType], meta: Map[String, String] = Map.empty)
+  def storeImage(bucket: String, id: String, file: File, mimeType: Option[MimeType],
```

Comment:
Not 100% on calling this `storeImage` when it's default argument means it doesn't.

Reply:
It's default argument means it *does*.

Reply:
huh, I'm sure it said `false`

Reply:
defaulting a boolean to true is confusing to be fair. I think this is cleaner now.